components/twitter/twitter.app.js

```diff
@@ -225,6 +225,18 @@ module.exports = {
         () => axios(config),
       );
     },
+    /**
+    * Enrich a Tweet object with ISO 8601 and timestamp (in milliseconds) representations of the `created_at` date/time, Tweet URL and user profile URL.
+    * @params {Object} tweet - An object representing a single Tweet as returned by Twitter's API
+    * @returns {Object} An enriched Tweet object is returned.
+    */
+    enrichTweet(tweet) {
+      tweet.created_at_timestamp = moment(tweet.created_at, 'ddd MMM DD HH:mm:ss Z YYYY').valueOf()
+      tweet.created_at_iso8601 = moment(tweet.created_at, 'ddd MMM DD HH:mm:ss Z YYYY').toISOString()
+      tweet.url = `https://twitter.com/${tweet.user.screen_name}/statuses/${tweet.id_str}`
+      if(tweet.user) { tweet.user.profile_url = `https://twitter.com/${tweet.user.screen_name}/` }
```

Comment:
we should set up a linter in this repo, but until then, just try to mimic the style in the rest of the code... space after `if` and if branch is on same line as conditional, don't use braces, or keep braces but just indent the branch in a new line.